src/main/java/com/tileman/TilemanModePlugin.java

```diff
@@ -587,6 +595,15 @@ private boolean containsAnyOf(MovementFlag[] comparisonFlags, MovementFlag[] fla
         return false;
     }
 
+    private boolean regionIsOnTutorialIsland(int regionId) {
+        for(int tutorialIslandRegionId: tutorialIslandRegionIds) {
```

Comment:
Not really an issue, but you should use a HashSet here for constant speed lookup, instead of having to iterate over the whole array.
Also if tutorialIslandRegionIds is a Set, then this whole function just becomes:
```
private boolean regionIsOnTutorialIsland(int regionId) {
  return tutIslandRegionIds.contains(regionId);
}
```

Don't need to fix it, just FYI

Reply:
Great suggestion, will fix that :)